src/pages/gen2/start/mobile-support/index.mdx

```diff
@@ -631,31 +631,33 @@ After the Amplify creation process, you can see a resource.ts file in the amplif
 The default code will create a Todo model with content and isDone property. The authorization rules below specify that owners, authenticated via your Auth resource can "create", "read", "update", and "delete" their own records.
 
 ```typescript
-import { type ClientSchema, a, defineData } from '@aws-amplify/backend';
+import { type ClientSchema, a, defineData } from "@aws-amplify/backend";
 
 const schema = a.schema({
   Todo: a
     .model({
       content: a.string(),
-      isDone: a.boolean()
+      isDone: a.boolean(),
+      createdAt: a.datetime(),
+      updatedAt: a.datetime(),
```

Comment:
these shouldn't be needed. They are autopopulated by Amplify Gen 2.

Reply:
It did not auto-populated for me and Elijah. Is it a bug then?